lib/modules/platform/bitbucket-server/schema.ts

```diff
@@ -35,6 +35,18 @@ export const PullRequestActivity = z.union([
 
 export type PullRequestActivity = z.infer<typeof PullRequestActivity>;
 
+export const ReviewerGroup = z.object({
+  name: z.string(),
+  users: z.array(UserSchema),
+  scope: z.object({
+    type: z.union([z.literal('REPOSITORY'), z.literal('PROJECT')]),
+  }),
+});
+
+export const ReviewerGroupsSchema = z.array(ReviewerGroup);
```

Comment:
```suggestion
export const ReviewerGroups = z.array(ReviewerGroup);
```

no trailing `Schema`

https://github.com/renovatebot/renovate/blob/main/docs/development/zod.md#use-schemats-files-for-zod-schemas